bubble_sort_by.rb

```diff
@@ -0,0 +1,28 @@
+def bubble_sort_by array
+    n=array.length
+    
+    loop do
+        move = false
+        aux= 0
```

Comment:
There should be space around operator; 
`n=array.length` ------Bad
`n = array.length` ------Good
One of the Microverse guideline state; students should make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide

_Kindly apply this to the rest the operator in your code._